### PR TITLE
Comment to fix old frontend queries that produce CC errors

### PR DESCRIPTION
**Pull request by lars18th**
Adds a comment to fix catalinii#1178

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,7 @@ Help
 	* [%] This symbol forces to read values as percentage
 	* [#] This symbol forces to read values as decibels
 	* eg: -M *:%1.0-#1.0 - not multiply the strength but force it as percentage and for the snr interpret it as decibels
+	* Zero values will disable drive polling, and this may solve CC errors with some hardware/drivers (eg: -M *:0-0)
```
Comment (Jalle19) [inline README.md:138]: Can you change the wording to "will disable frontend polling"?

**Comment by madmanxxx** [comment]
Apart from the typo ("_drive_ polling") IMO it should be "frontend polling"

**Comment by catalinii** [comment]
This should be part of minisatip.c

**Comment by Jalle19** [comment]
That too. Honestly I think we should remove the usage information from the README, it's hard to keep it up to date and some options are platform and/or feature-specific anyway.

**Comment by lars18th** [comment]
> That too. Honestly I think we should remove the usage information from the README, it's hard to keep it up to date and some options are platform and/or feature-specific anyway.

I don't agree with that because I use this page a lot.

File: README.md (hunk 1 above)
Comment (lars18th) [reply README.md]: Done

**Comment by Jalle19** [comment]
I use it too

**Comment by lars18th** [comment]
Please, in any case it will be useful to merge this PR now.

**Pull request merged**